tests/commands/convert/test_build_gui_context.py

```diff
@@ -1,5 +1,5 @@
-def test_overrides_are_not_used(convert_command):
-    overrides = {"gui_framework": "Toga"}
+def test_overrides_are_used(convert_command):
+    overrides = {"gui_framework": "Toga", "console_app": "Console"}
     out = convert_command.build_gui_context({}, overrides)
-    assert out == {"gui_framework": "None"}
-    assert overrides == {"gui_framework": "Toga"}
+    assert out["gui_framework"] == "None"
+    assert overrides["gui_framework"] == "Toga"
```

Comment:
This changes the test quite a bit - it goes from checking the *entire* contents of the dictionary, to checking one specific field in the dictionary. If the method adds additional fields, the test won't raise this as a failure.